src/api.js

```diff
@@ -1,19 +1,29 @@
 import axios from "axios";
 
-const allArticles = () => {
+const getAllArticles = () => {
   return axios
     .get("https://nc-news-project-q2e5.onrender.com/api/articles")
     .then((response) => {
       return response;
     });
 };
 
-const articleById = (article_id) => {
+const getArticleById = (article_id) => {
   return axios
     .get(`https://nc-news-project-q2e5.onrender.com/api/articles/${article_id}`)
     .then((response) => {
       return response;
     });
 };
 
-export { allArticles, articleById };
+const getCommentsByArticleId = (article_id) => {
+  return axios
+    .get(
+      `https://nc-news-project-q2e5.onrender.com/api/articles/${article_id}/comments`
+    )
+    .then((response) => {
+      return response;
```

Comment:
We can directly return the comments here as the function name suggests we are doing rather then needing to do this after invoking the function